src/schema/v2/marketingCollections.ts

```diff
@@ -0,0 +1,180 @@
+import { MarketingCollectionsSorts } from "./sorts/marketingCollectionsSort"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "schema/v2/fields/pagination"
+import { SlugAndInternalIDFields, NodeInterface } from "./object_identification"
+
+export const MarketingCollectionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "MarketingCollection",
+  interfaces: [NodeInterface],
+  fields: {
+    ...SlugAndInternalIDFields,
+    description: {
+      type: GraphQLString,
+      resolve: ({ description }) => description,
+    },
+    descriptionMarkdown: {
+      type: GraphQLString,
+      resolve: ({ description_markdown }) => description_markdown,
+    },
+    credit: {
+      type: GraphQLString,
+      resolve: ({ credit }) => credit,
+    },
+    category: {
+      type: GraphQLString,
+      resolve: ({ category }) => category,
+    },
+    priceGuidance: {
+      type: GraphQLString,
+      resolve: ({ price_guidance }) => price_guidance,
+    },
+    published: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ published }) => published,
+    },
+    title: {
+      type: GraphQLString,
+      resolve: ({ title }) => title,
+    },
+    geneIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ gene_ids }) => gene_ids,
+    },
+    artistIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ artist_ids }) => artist_ids,
+    },
+    keyword: {
+      type: GraphQLString,
+      resolve: ({ keyword }) => keyword,
+    },
+    isFeaturedArtistContent: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ is_featured_artist_content }) => is_featured_artist_content,
+    },
+    featuredArtistExclusionIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ featured_artist_exclusion_ids }) =>
+        featured_artist_exclusion_ids,
+    },
+    headerImageId: {
+      type: GraphQLString,
+      resolve: ({ header_image_id }) => header_image_id,
+    },
+    thumbnailId: {
+      type: GraphQLString,
+      resolve: ({ thumbnail_id }) => thumbnail_id,
+    },
+    showHeaderArtworksRail: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ show_header_artworks_rail }) => show_header_artworks_rail,
+    },
+    showFeaturedArtists: {
+      type: GraphQLNonNull(GraphQLBoolean),
+      resolve: ({ show_featured_artists }) => show_featured_artists,
+    },
+    artworkIds: {
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ artwork_ids }) => artwork_ids,
+    },
+  },
+})
+
+export const MarketingCollection: GraphQLFieldConfig<void, ResolverContext> = {
+  type: MarketingCollectionType,
+  description: "Marketing Collection",
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The slug or ID of the Marketing Collection",
+    },
+  },
+  resolve: (_root, { id, size }, { marketingCollectionLoader }) => {
+    if (!marketingCollectionLoader)
```

Comment:
You don't need to be authenticated to retrieve a marketing collection (otherwise how would logged-out users see the collection page).